service/src/test/java/uk/gov/hmcts/reform/fpl/enums/UserRolesTest.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserRolesTest {
+
+    @ParameterizedTest
+    @EnumSource(UserRole.class)
+    public void shouldProduceAllIdamRoles(UserRole userRole) {
+        assertThat(userRole.getRoles()).containsExactly("caseworker", "caseworker-publiclaw", userRole.getRole());
```

Comment:
Would it be better to pass this the role hardcoded as well? Instead of calling userRole.getRole(), just in case that gets accidentally changed.

Reply:
agree with you, but most people dont like such approach (e.g using string constants from prod test in test code)
changed